readme.md

```diff
@@ -8,16 +8,87 @@ This is a telegram bot which can:
 Future feature plans:
 - FA notifications
 
-## Setup
+You can use it freely at [@FASearchBot](https://t.me/FASearchBot), add it to your own chats and channels, or use it inline anywhere by mentioning it
+
+## Commands
+
+### Neaten
+
+Private message it a link to a Fur Affinity submission page, thumbnail url or image download url, i.e, a link beginning with `http://?.facdn.net/art/` and it will reply with the image's thumbnail and a link to that image's gallery page.
+
+### Inline Responses
+
+In any chat, mention the bot and pass it an FA query
+`@FASearchBot [query]` and pick from the inline results which image you'd like to share.
+
+You can also use these qualifiers inline:
+`@FASearchBot gallery:username`
+`@FASearchBot scraps:username`
+`@FASearchBot favs:username`
+
+### Subscriptions
+
+`/add_subscription [query]`
+`/list_subscriptions`
+`/remove subscription [query]`
+
+`/add_blocklisted_tag [tag]`
+`/list_blocklisted_tag [tag]`
+`/remove_blocklisted_tag [tag]`
+
+### Search Queries
+
+The bot will support all Fur Affinity queries, and some extra filter options:
+
+Taken from FurAffinity's search documentation: 
+> Search understands basic boolean operators:
+> * AND: `hello & world`
+> * OR : `hello | world`
+> * NOT: `hello -world` -or- `hello !world`
+> 
+> Grouping: `(hello world)`
+> Example: `( cat -dog ) | ( cat -mouse)`
+>
+> *Capabilities*
+> Field searching: `@title hello @message world`
+> Phrase searching: `"hello world"`
+> Word proximity searching: `"hello world"~10`
+> Quorum matching: `"the world is a wonderful place"/3`
+> Example: `"hello world" @title "example program"~5 @message python -(php|perl)`
+> 
+> Available Fields: `@title` `@message` `@filename` `@lower` (artist name as it appears in their userpage URL) `@keywords`
+> Example: `fender @title fender -dragoneer -ferrox @message -rednef -dragoneer`
+
+### Extra filters provided by the bot
+
+`rating:general` (or `rating:safe`)
+`rating:mature` (or `rating:questionable`)
+`rating:adult` (or `rating:explicit`)
+
+You can also use `-` or `!` to exclude from the results, e.g. `-rating:explicit`, `!rating:safe`
+
+### Miscellaneous
+
+`/beep` - responds with `boop`
+
+Send or forward the bot an image with no text as a private message, and it will recommend you try [@foxbot](https://t.me/foxbot) or [@FindFurryPicBot](https://t.me/FindFurryPicBot) 
+
+## Development 
+
+### Running the bot for yourself
+
 To set this up, you'll need to:
+
  - Download or clone the repo
  - Install requirements: `pip install -r requirements.txt`
  - Add a `config.json` in the base directory, like so:
+
 ```json
 {
   "bot_key": "---",
   "api_url": "https://faexport.boothale.net/"
```

Comment:
Probably worth updating this link to `https://faexport.spangle.org.uk` now, boothale's version is broken for most endpoints, and he's been MIA long enough that his telegram has now been deleted :(